test/Sentry.Maui.Tests/SentryMauiOptionsTests.cs

```diff
@@ -56,4 +56,30 @@ public void AttachScreenshots_Default()
         var options = new SentryMauiOptions();
         Assert.False(options.AttachScreenshot);
     }
+    [Fact]
```

Comment:
nit:
we add lines between test
```suggestion

    [Fact]
```